### DISCUSSION TITLE
太好用导航，希望加个自动侧边栏功能

### DISCUSSION THREAD

**Discussion by wbvi** [Yiov/vitepress-doc#5] [Q&A] [answered]
茂神您好，很感谢您开源啦这个，我不会开发，希望能实现自动的侧边栏功能。

**Comment by Yiov** [comment, marked as answer]
我不是茂茂大神哈，本尊在这
https://github.com/maomao1996/vitepress-nav-template

文章自动侧边栏可以参考
https://github.com/QC2168/vite-plugin-vitepress-auto-sidebar

**Reply by wbvi** [reply]
谢谢
